cli/cook/subcommands/config.py

```diff
@@ -0,0 +1,118 @@
+from cook.util import print_info
+
+from cook import configuration, colors
+from cook.configuration import load_config
+
+
+def get_in(dct, keys):
+    """Gets the value in dct at the nested path indicated by keys"""
+    for key in keys:
+        if not isinstance(dct, dict):
+            return None
+        try:
+            dct = dct[key]
+        except KeyError:
+            return None
```

Comment:
Is there a reason to prefer the `try`/`except` here rather than using `dct.get(key)`?

Reply:
Switched to checking if `key in dct`